tests/test_core.py

```diff
@@ -2578,6 +2578,15 @@ def test_pthread_proxying(self):
     self.do_run_in_out_file_test('pthread/test_pthread_proxying.c',
                                  emcc_args=args, interleaved_output=False)
 
+  @node_pthreads
+  def test_pthread_proxying_cpp(self):
+    self.set_setting('EXIT_RUNTIME')
+    self.set_setting('PROXY_TO_PTHREAD')
+    self.set_setting('INITIAL_MEMORY=32mb')
+    args = [f'-I{path_from_root("system/lib/pthread")}']
```

Comment:
We should probably install this header in the sysroot if we want users to be able to use it.  Are we ready for that yet or should we keep it internal for now?

Reply:
Yeah, we're probably ready for that. I can make that move (and add documentation) as a follow-up.